scripts/validate-packages.py

```diff
@@ -125,15 +135,25 @@ def _validate_revision(given_package, revision, path):
         eprint("\tOK")
 
     # Validate that we don't drop information during the conversion
-    oldPackage = LooseVersion(
+    old_package = LooseVersion(
         package_json.get('minDcosReleaseVersion', "1.0")) < LooseVersion("1.8")
-    if (oldPackage and resource_json and 'cli' in resource_json and
+    if (old_package and resource_json and 'cli' in resource_json and
             command_json is None):
         sys.exit('\tERROR\n\nA package with CLI specified in resource.json is '
                  'only supported when minDcosReleaseVersion is greater than '
                  '1.8.')
 
 
+def _validate_package_with_directory(given_package, actual_package_name):
+    if(not PACKAGE_FOLDER_PATTERN.match(given_package)
+       or given_package != actual_package_name):
```

Comment:
Let's have a separate check and error message for each condition, again providing helpful information for debugging.